python/afdko/buildcff2vf.py

```diff
@@ -156,14 +152,12 @@ def make_curve_coords(self, coords, is_default):
         if is_default:
             new_coords = []
             for i, cur_coords in enumerate(coords):
-                prev_coords = prev_cmd[1][i]
-                master_coords = self.make_flat_curve(prev_coords[:2],
-                                                     cur_coords)
+                master_coords = self.make_flat_curve(cur_coords)
                 new_coords.append(master_coords)
         else:
             cur_coords = coords
             prev_coords = prev_cmd[1][-1]
```

Comment:
You're not using `prev_coords`, so this line can be removed (as can line 151 above).